bin/steps/swift-build

```diff
@@ -1,2 +1,2 @@
 puts-step "Building package ($SWIFT_BUILD_CONFIGURATION configuration)"
-swift build $SWIFT_BUILD_FLAGS -c "$SWIFT_BUILD_CONFIGURATION"
+swift build $SWIFT_BUILD_FLAGS -c "$SWIFT_BUILD_CONFIGURATION" --static-swift-stdlib
```

Comment:
We should also use jemalloc but that should be another PR.
It shouldn't be as problematic.

Reply:
Good idea, actually.

Reply:
That integration will more likely come in the form of a section in the README, as installing jemalloc cleanly is a solved problem: https://github.com/brian-kephart/heroku-buildpack-jemalloc.